ddtrace/tracer/tracer.go

```diff
@@ -296,6 +313,17 @@ func (t *tracer) StartSpan(operationName string, options ...ddtrace.StartSpanOpt
 			// this is a root span in the global service; runtime metrics should
 			// be linked to it:
 			span.setMeta("language", "go")
+			// set app tags on the root span
```

Comment:
App tags have nothing to do with runtime metrics. Why did you put them in here? They need to go before this `if` clause, outside of it.

Reply:
From the comments, it looked as if it was root inside this if.